cpp/src/groupby/hash/groupby.cu

```diff
@@ -502,7 +502,7 @@ void compute_single_pass_aggs(table_view const& keys,
   bool skip_key_rows_with_nulls = keys_have_nulls and include_null_keys == null_policy::EXCLUDE;
 
   auto row_bitmask =
-    skip_key_rows_with_nulls ? cudf::detail::bitmask_and(keys, stream) : rmm::device_buffer{};
+    skip_key_rows_with_nulls ? cudf::detail::bitmask_and(keys, stream).mask : rmm::device_buffer{};
```

Comment:
Should the `rmm::device_buffer` in the "else" of this ternary statement be constructed with a stream and/or memory resource?

Reply:
It's not needed. Default ctor creating empty buffer takes no argument: [here](https://github.com/rapidsai/rmm/blob/branch-21.12/include/rmm/device_buffer.hpp#L84-L91)